src/Robo/Tasks/Drush.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Acquia\Blt\Robo\Tasks;
+
+use Robo\Task\CommandStack;
+use Robo\Contract\VerbosityThresholdInterface;
+use Robo\Common\CommandArguments;
+
+/**
+ * Runs Drush commands in stack. You can use `stopOnFail()` to point that stack should be terminated on first fail.
+ *
+ * ``` php
+ * $this->taskDrush()
+ *     ->drush('updb')
+ *     ->drush('cr')
+ *     ->run();
+ * ```
+ */
+class Drush extends CommandStack
+{
+  use CommandArguments;
+
+  /**
+   * Site alias to prepend to each command.
+   *
+   * @var string
+   */
+  private $alias;
```

Comment:
I prefer to default to protected unless there's a particular reason for private.